src/prepare-build.ts

```diff
@@ -26,16 +32,20 @@ export async function buildOptions(inputs: Inputs) {
 	const flags = [];
 	flags.push(...getFailOnFlags(toolchain, inputs.BUILD_FAIL_ON));
 
-	return { toolchain, source, destination, flags, configOverride };
+	return { toolchain, source, destination, gitRevision, flags, configOverride };
```

Comment:
I think best would be add the `gitRevision` to each config override above, instead of setting it later. 
Does Bikeshed set it automatically in build process, or we can pass it via config also? If not passable, can pass conditionally for ReSpec only.

Reply:
bikeshed does it automatically already on its own

Reply:
can you say a bit more why you think it should be in the override? this feels typically like a setting that shouldn't need to be overridden by a static configuration file

Reply:
I was suggesting that only because as we can to pass it down less manually that way, as `configOverride` will get passed to ReSpec as `respecConfig` override (via URL parameters).
https://github.com/w3c/spec-prod/pull/167/files#diff-f065c431b4f45758b2e047fcd38135fdcb2327564def7a33ab41e84898210fc8L95-L101

We use same method for setting `publishDate`: https://github.com/w3c/spec-prod/blob/958b6d59f51720a7b0f2d402f87ba193e39bc080/src/prepare-build.ts#L139-L140

Reply:
(I'll refactor this part someday, I find it confusing too)

Reply:
I think if you can add following above and reset `src/build.ts`, that'd be enough:
```ts
if (toolchain === 'respec) {
  configOverride.gh.gitRevistion = configOverride.w3c.gitRevision = gitRevision;
}
```

Reply:
If not, feel free to do it what you find simpler. We can refactor it eventually.